tests/system/datastore/test_v3io.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import random
+import subprocess
+
+import pytest
+
+import mlrun.datastore
+from tests.system.base import TestMLRunSystem
+
+
+@TestMLRunSystem.skip_test_if_env_not_configured
+@pytest.mark.enterprise
+class TestV3ioDataStore(TestMLRunSystem):
+    @staticmethod
+    def _skip_set_environment():
+        return True
+
+    def test_v3io_large_object(self):
+
+        with open("tempfile_1", "wb") as f:
+            file_size = 20 * 1024 * 1024  # 20MB
```

Comment:
So it seems to me that this doesn't test the new code, because it's under the 1GB limit. While I wouldn't expect thata we upload such a large file in the test, how about making `max_chunk_length` an optional parameter so that we can override it and test the functionality? Then we can parameterize this test with `max_chunk_length=None` and `max_chunk_length=1024*1024`, or something like that.

Reply:
fixed (added upload_helper() function with ability to set max_chunk_length)